### PR TITLE
Fix docstrings: Conform to numpydoc style

### PR DESCRIPTION
**Pull request by andersy005**
The previously used docstring style resulted in sphinx docs pages that weren't nicely formatted. This PR is an attempt to conform the docstrings to the numpydoc style: https://numpydoc.readthedocs.io/en/latest/format.html. This PR enables napoleon sphinx extension which produces a well formatted API reference page:

Screen Shot 2020-07-29 at 11 47 49 AM

### CODE REVIEW

File: ldcpy/util.py [hunk 1]
```diff
@@ -54,31 +53,29 @@ def print_stats(ds, varname, set1, set2, time=0, sig_dig=4):
...
+        The variable of interest in the dataset
+    set1 : str
+        The collection label of the "control" data
+    set2 : str
+        The collection label of the (1st) data to compare
+    time : int, optional
+        The time index used to compare the two netCDF files (default 0)
+
+    sig_dig : int, optional
+        (default 4)
```
Comment (andersy005) [inline ldcpy/util.py:70]: I wasn't sure what the description for `sig_dig` should be. What does `sig_dig` mean? ;)

Comment (pinarda) [reply ldcpy/util.py:70]: Oops, that should be the number of significant digits to print for each metric, but it's actually one less than that. I just pushed a change to fix the problem. Thanks!

Comment (andersy005) [reply ldcpy/util.py]: Great... Just merged latest changes from master into this PR. Feel free to merge this into master once the CI is passing.

**Comment by andersy005** [comment]
> The formatting of the **kwargs parameter in open_datasets() is messed up in the first screenshot you posted. The rest looks really nice!

The latest commit fixes this
 
Screen Shot 2020-07-29 at 3 43 32 PM

**Pull request merged**